index.html

```diff
@@ -292,7 +292,7 @@ <h3>
               </td>
             </tr>
             <tr>
-              <td><a href="https://json-web-proofs.github.io/json-web-proofs/">JSON Web Proof (JWP)</a></td>
+              <td><a href="https://json-web-proofs.github.io/json-web-proofs/">VC-JSON Web Proof (JWP)</a></td>
```

Comment:
@msporny @brentzundel -- Should `VC-JSON Web Proof (JWP)` not have become `VC-JSON Web Proof (VC-JWP)` (changing the parenthetical), or maybe `Verifiable Credential JSON Web Proof (VC-JWP)` (changing the parenthetical *and* expanding VC in the full name)?